### PR TITLE
added linkouts queries/resolvers for GeneFamily and PanGeneSet

### PR DESCRIPTION
**Pull request by adf-ncgr**
depends on https://github.com/legumeinfo/microservices/pull/613

### CODE REVIEW

**Comment by sammyjava** [comment]
I'll hold off on reviewing this until https://github.com/legumeinfo/microservices/pull/613 is merged.

File: src/resolvers/intermine/gene-family.ts [hunk 1]
```diff
@@ -48,5 +51,10 @@ ResolverMap => ({
                 // @ts-ignore: implicit type any error
                 .then(({data: results}) => results);
         },
+        linkouts: async (geneFamily, _, { dataSources }) => {
+            const {identifier} = geneFamily;
+            console.log("gonna call it with identifier");
```
Comment (alancleary) [inline src/resolvers/intermine/gene-family.ts:56]: Remove this!

**Review by alancleary — CHANGES_REQUESTED** [review]
I can't actually run an integration test with the linkout microservice but everything else looks good. The only issue I see is an errant `console.log`.

File: src/resolvers/intermine/gene-family.ts (hunk 1 above)
Comment (adf-ncgr) [reply src/resolvers/intermine/gene-family.ts]: eagle eyes!
If you like, we can wait to approve until I've deployed the new linkouts so as to do the proper testing

Comment (adf-ncgr) [reply src/resolvers/intermine/gene-family.ts]: OK, the new microservices deployment is complete, so the new services should return data for gene families with prefix legfed_v1_0 and pangenes with prefix Glycine.pan4 (more to be added, but that should suffice for the rigors of testing)

**Review by alancleary — APPROVED** [review]
Confirmed working with the updated linkouts microservice.

**Pull request merged**